frontend/docs/pages/home/features/timeouts.mdx

```diff
@@ -140,6 +154,26 @@ const myStep: CreateStep<any, any> = {
   },
 };
 ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+```go
+cleanup, err := run(events, worker.WorkflowJob{
```

Comment:
I'm not sure where this `run` method is defined?

Reply:
yeah, I think it was there in the user guide in this page and I meant to change it but might've missed it. Will fix this